src/CONST.ts

```diff
@@ -1254,6 +1254,7 @@ const CONST = {
     PUSHER: {
         PRIVATE_USER_CHANNEL_PREFIX: 'private-encrypted-user-accountID-',
         PRIVATE_REPORT_CHANNEL_PREFIX: 'private-report-reportID-',
+        PRESENT_ACTIVE_GUIDES: 'presence-activeGuides',
```

Comment:
presence and private are reserved keywords for pusher channel. Let's use them
```suggestion
        PRESENCE_ACTIVE_GUIDES: 'presence-activeGuides',
```